justfile

```diff
@@ -57,6 +57,9 @@ reseed:
 import-hn:
 	npx tsx import-hn.ts ./other/hn-data/*.json.gz
 
+import-society-library:
+	cd other && npx tsx import-society-library-debatemap.ts society-library-openai-debatemap.json
```

Comment:
don't hardcode the json file here, instead leave it out and pass it as an argument to the just command